### ISSUE TITLE
Can interpolation image have different seed?

### ISSUE THREAD

**Issue by edi2213** [DiceOwl/StableDiffusionStuff#7]
It's possible that first image(0) and the last one(1) have different seed?

**Comment by DiceOwl** [comment]
Yes, check "Interpolate Varseed", enable "Extra" at seed settings, set "variation strength"=1, and input the second seed into "variation seed"

**Comment by edi2213** [comment]
Thanks for your response. I try but doesn't work for me. I do something wrong?
Here is my settings:
![image](https://github.com/DiceOwl/StableDiffusionStuff/assets/133267489/30713332-960a-41d8-9ba1-74cc96a7b356)

**Comment by edi2213** [comment]
![image](https://github.com/DiceOwl/StableDiffusionStuff/assets/133267489/d83b1a6b-c191-47d9-aa5d-3798e17f3933)

**Comment by DiceOwl** [comment]
I tested it myself, and you are right, it no longer works as intended. I suspect something changed in the way auto1111 manages the noise. However, right now I do not have a work around for this problem.
